pom.xml

```diff
@@ -70,12 +71,14 @@ See the Mulan PSL v2 for more details.
     <maven.compiler.target>${java.version}</maven.compiler.target>
 
     <commons-lang.version>3.12.0</commons-lang.version>
+    <commons-collection.version>4.4</commons-collection.version>
     <commons-codec.version>1.15</commons-codec.version>
-    <netty.version>4.1.68.Final</netty.version>
+    <guava.version>31.0.1-jre</guava.version>
```

Comment:
Is guava necessary here? If we introduce it, there may be more compatibility issues, as it does not guarantee forward compatibility.